sympy/assumptions/assume.py

```diff
@@ -301,6 +302,7 @@ class Predicate(Boolean, metaclass=PredicateMeta):
     """
 
     is_Atom = True
+    _arity: Any = None
```

Comment:
Why is this `Any`?

Reply:
It the arity attribute can either be none or an integer so it cannot have a set type.

Reply:
That means it should be int | None